using_images/s2i_images/perl.adoc

```diff
@@ -25,35 +25,52 @@ new image containing your Perl application. This resulting image can be run
 either by {product-title} or by Docker.
 
 == Versions
-Currently, {product-title} supports version
-https://github.com/openshift/sti-perl/tree/master/5.16[5.16] of Perl.
+Currently, {product-title} supports versions
+https://github.com/openshift/sti-perl/tree/master/5.16[5.16],
+https://github.com/openshift/sti-perl/tree/master/5.20[5.20], and
+https://github.com/openshift/sti-perl/tree/master/5.24[5.24] of Perl.
 
 == Images
 
-This image comes in two flavors, depending on your needs:
+ifdef::openshift-online[]
+RHEL 7 images are available through Red Hat's subscription registry:
```

Comment:
s/Red Hat's subscription registry/the Red Hat Registry/

See https://access.redhat.com/containers/#/faq.